src/azure-cli-core/setup.py

```diff
@@ -8,7 +8,7 @@
 from codecs import open
 from setuptools import setup, find_packages
 
-VERSION = "2.34.0"
+VERSION = "2.34.1"
```

Comment:
Please do not upgrade CLI version manually. It will be automatically generated by CI pipeline when releasing